plugins/modules/virt_volume.py

```diff
@@ -0,0 +1,711 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Maciej Delmanowski <drybjed@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: virt_volume
+author:
+    - Leonardo Galli (@galli-leo)
+short_description: Manage libvirt volumes inside a storage pool
+description:
+    - Manage I(libvirt) volumes inside a storage pool.
+options:
+    name:
+        aliases: [ "volume" ]
+        description:
+            - Name of the volume being managed. Note that the volume must be previously
+              defined with xml.
```

Comment:
```suggestion
            - Name of the volume being managed. Note that the volume must be previously
              defined with xml.
```
Is it not required? Just to be sure

Reply:
It is required, with the exception of the `list_volumes`, `info` and `facts` command.